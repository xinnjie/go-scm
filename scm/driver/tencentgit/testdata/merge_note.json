{
  "id": 1383976,
  "parent_id": null,
  "body": "test comment",
  "attachment": null,
  "author": {
    "id": 176405,
    "username": "xinnjie",
    "web_url": "https://git.code.tencent.com/u/xinnjie",
    "name": "xinnjie",
    "state": "active",
    "avatar_url": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
  },
  "created_at": "2021-08-24T11:41:23+0000",
  "system": false,
  "line_code": null,
  "file_path": null,
  "comments": []
}