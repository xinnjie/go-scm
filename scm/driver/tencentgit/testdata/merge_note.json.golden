{
  "ID": 1383976,
  "Body": "test comment",
  "Author": {
    "Login": "xinnjie",
    "Name": "xinnjie",
    "Email": "",
    "Avatar": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
  },
  "Created": "2021-08-24T11:41:23Z",
  "Updated": "2021-08-24T11:41:23Z"
}